src/crypto/index.js

```diff
@@ -276,6 +281,60 @@ Crypto.prototype.init = async function() {
     this._checkAndStartKeyBackup();
 };
 
+/**
+ * Bootstrap Secure Secret Storage if needed by creating a default key and signing it with
+ * the cross-signing master key. If everything is already set up, then no
+ * changes are made, so this is safe to run to ensure secret storage is ready
+ * for use.
+ *
+ * @param {function} [opts.doInteractiveAuthFlow] Optional. Function called to
+ * await an interactive auth request.
+ * Args:
+ *     {function} A function that makes the request requiring auth. Receives the
+ *     auth data as an object.
+ */
+Crypto.prototype.bootstrapSecretStorage = async function({
+    doInteractiveAuthFlow,
+} = {}) {
+    logger.log("Bootstrapping Secure Secret Storage");
+
+    // Create cross-signing keys if they don't exist, as we want to sign the SSSS default
+    // key with the cross-signing master key. The cross-signing master keys is also used
+    // to verify the signature on the SSSS default key when adding secrets, so we
+    // effectively need it for both reading and writing secrets.
+    let crossSigningKeysChanged = false;
+    if (!this._crossSigningInfo.getId()) {
+        logger.log("Cross-signing keys not found, creating new keys");
```

Comment:
We should also check if there are cross-signing keys for this account on the server but we haven't confirmed them and stored them locally, in which case we need to get the private keys.

Reply:
I have added a path to get the cross-signing private keys from secret storage if they exist.